libs/dh/market-participant/actors/feature-actors/src/lib/drawer/balance-responsible-relation-tab/dh-balance-responsible-relation-tab.component.ts

```diff
@@ -246,4 +251,41 @@ export class DhBalanceResponsibleRelationTabComponent {
       );
     }
   }
+
+  download() {
+    const balanceResponsibleRelations = this.balanceResponsibleRelationsRaw();
+
+    if (!balanceResponsibleRelations) {
+      return;
+    }
+
+    const columnsPath =
+      'marketParticipant.actorsOverview.drawer.tabs.balanceResponsibleRelation.columns';
+
+    const headers = [
+      `"${translate(columnsPath + '.balanceResponsibleId')}"`,
+      `"${translate(columnsPath + '.balanceResponsibleName')}"`,
+      `"${translate(columnsPath + '.energySupplierId')}"`,
+      `"${translate(columnsPath + '.energySupplierName')}"`,
+      `"${translate(columnsPath + '.gridAreaId')}"`,
+      `"${translate(columnsPath + '.meteringPointType')}"`,
+      `"${translate(columnsPath + '.status')}"`,
+      `"${translate(columnsPath + '.start')}"`,
+      `"${translate(columnsPath + '.end')}"`,
+    ];
+
+    const lines = balanceResponsibleRelations.map((balanceResponsibleRelation) => [
+      `"${balanceResponsibleRelation.balanceResponsibleWithName?.id}"`,
+      `"${balanceResponsibleRelation.balanceResponsibleWithName?.actorName.value}"`,
+      `"${balanceResponsibleRelation.energySupplierWithName?.id}"`,
+      `"${balanceResponsibleRelation.energySupplierWithName?.actorName.value}"`,
+      `"${balanceResponsibleRelation.gridAreaId}"`,
+      `"${balanceResponsibleRelation.meteringPointType}"`,
+      `"${balanceResponsibleRelation.status}"`,
+      `"${balanceResponsibleRelation.validPeriod.start}"`,
+      `"${balanceResponsibleRelation.validPeriod.end}"`,
```

Comment:
suggestion: some of these properties can be `undefined`/`null`. In such case, what will the output in the CSV file be? `"undefined"`/`"null"`? If that's the case, let's set it to en empty string